ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -227,13 +227,27 @@ open class BadgeView: UIView, TokenizedControlInternal {
         backgroundView.frame = bounds
 
         if let customViewSize = customViewSize(for: frame.size), customViewSize != .zero {
-            let customViewOrigin = CGPoint(x: customViewPadding.left, y: (frame.height - customViewSize.height) / 2)
-            dataSource?.customView?.frame = CGRect(origin: customViewOrigin, size: customViewSize)
-            let labelOrigin = CGPoint(x: customViewPadding.left + customViewPadding.right + customViewSize.width, y: (frame.height - labelSize.height) / 2)
-            let labelSizeThatFits = CGSize(width: frame.size.width - labelOrigin.x, height: labelSize.height)
+            // H:|-(cvp.left)-[customView]-(cvp.right)-[label]-(lhp)-|
+            let customViewPadding = customViewPadding
```

Comment:
This is just for perf right? Fetch customViewPadding once?

Reply:
Exactly. The custom getter for `customViewPadding` does a bit of work, so no need to run it more than once here.

Reply:
Same with `let labelSize = labelSize` below.